### PR TITLE
Modify content (Write Example)

### PR DESCRIPTION
**Pull request by HalynaDumych**
Modified the content of the `dataservice-write-example.md` file
and aligned it with the Developer's Guide.

Relates-to: OLPEDGE-1146

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: docs/dataservice-write-example.md [hunk 1]
```diff
@@ -1,213 +1,281 @@
...
 
-### Prerequisites
+After building and running the example, the following message displays automatically: "Publish Successful - TraceID: \<TraceId generated by the platform>".
 
-* Set up the Android environment.
-* Replace the placeholders in `examples/dataservice-write/example.cpp` with your app access key id, secret access key, catalog HRN and layer.
+## Build and Run on Android
 
-### Build HERE OLP SDK for C++
+To integrate the HERE OLP SDK for C++ libraries in the Android project:
```
Comment (borzun) [inline docs/dataservice-write-example.md]:
```suggestion
To integrate the HERE OLP SDK for C++ libraries in the Android example project:
```

File: docs/dataservice-write-example.md [hunk 2]
```diff
@@ -1,213 +1,281 @@
...
-* Configure the SDK with `OLP_SDK_BUILD_EXAMPLES` set to `ON`.
-* Specify the path to Android NDK's toolchain file set via the `CMAKE_TOOLCHAIN_FILE` variable.
-* In case you want to build SDK for a specific Android platform, use `-DANDROID_PLATFORM CMake` flag, and `-DANDROID_ABI`, if you want to build for specific Android architecture. For more details, see [NDK-specific CMake variables](https://developer.android.com/ndk/guides/cmake#variables).
+### <a name="prerequisites-android"></a>Prerequisites
 
-```bash
-mkdir build && cd build
-cmake .. -DOLP_SDK_BUILD_EXAMPLES=ON -DCMAKE_TOOLCHAIN_FILE=$NDK_ROOT/build/cmake/android.toolchain.cmake -DANDROID_ABI=arm64-v8a
-```
+Before you integrate the HERE OLP SDK for C++ libraries in your Android project:
```
Comment (borzun) [inline docs/dataservice-write-example.md]:
```suggestion
Before you integrate the HERE OLP SDK for C++ libraries in the Android example project:
```

File: docs/dataservice-write-example.md [hunk 3]
```diff
@@ -1,213 +1,281 @@
...
+
+1. In the Android Studio IDE, open the `build/examples/dataservice-write/android/build.gradle` script.
+2. Provide your application access key ID, access key secret, catalog HRN, and layer name.
+3. Install and run the `dataservice_write_example` APK.
 
-This example shows how to integrate and use the HERE OLP SDK for C++ for a basic iOS application written in the Objective-C language.
+The main screen displays the following message: "Publish Successful".
 
-### Prerequisites
+If you encounter an error, in your device LogCat, find the error message.
```
Comment (borzun) [inline docs/dataservice-write-example.md]:
```suggestion
If you encounter an error, find the error message in your device Logcat.
```
Refer to my previous comment in other PR: https://github.com/heremaps/here-olp-sdk-cpp/pull/529/files#r361130896

File: docs/dataservice-write-example.md [hunk 4]
```diff
@@ -1,213 +1,281 @@
...
 
-* Set up the iOS development environment > install the `XCode` and command-line tools.
-* Install external dependencies > refer to the `README.md` file, located under the `<olp-sdk-root>/README.md`.
-* Replace the placeholders in `examples/dataservice-write/example.cpp` with your app access key id, secret access key, catalog HRN and layer.
+## Building and running on iOS
+
+To integrate the HERE OLP SDK for C++ libraries in a basic iOS application written in the Objective-C language:
```
Comment (borzun) [inline docs/dataservice-write-example.md]:
```suggestion
To integrate the HERE OLP SDK for C++ libraries in the iOS example application written in the Objective-C language:
```

File: docs/dataservice-write-example.md (hunk 1 above)
Comment (HalynaDumych) [reply docs/dataservice-write-example.md]: Fixed

File: docs/dataservice-write-example.md (hunk 2 above)
Comment (HalynaDumych) [reply docs/dataservice-write-example.md]: Fixed

File: docs/dataservice-write-example.md (hunk 3 above)
Comment (HalynaDumych) [reply docs/dataservice-write-example.md]: Deleted

File: docs/dataservice-write-example.md (hunk 4 above)
Comment (HalynaDumych) [reply docs/dataservice-write-example.md]: Fixed

**Pull request merged**